esma.cmake

```diff
@@ -83,6 +83,7 @@ option (ESMA_ALLOW_DEPRECATED "suppress warnings about deprecated features" ON)
 # Unit testing
 option (PFUNIT "Activate pfunit based tests" OFF)
 if (PFUNIT)
+   find_package(PFUNIT REQUIRED)
```

Comment:
I agree with Matt's suggestion.     That's the way I do it in my other projects.   

We do want to make certain that none of the tests are added to the "all" target though, as everyone will have PFUNIT found via baselibs going forward and not everyone wants to build the tests, and not everyone would even understand what they are for.